lib/osut/utils.rb

```diff
@@ -1871,12 +1872,14 @@ def addSubs(model = nil, s = nil, subs = [], clear = false, bfr = 0.005)
 
     # --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- #
     # Log/reset (or abandon) conflicting user-set geometry key:value pairs:
-    #   - sub[:head      ] # e.g. std 80" door + frame/buffers
-    #   - sub[:sill      ] # e.g. std 30" sill + frame/buffers
-    #   - sub[:height    ] # any sub surface height below "head"
-    #   - sub[:width     ] # e.g. 1.200 m
-    #   - sub[:offset    ] # centreline-to-centreline between subs (if array)
-    #   - sub[:centreline] # left/right of base surface centreline, e.g. -0.2 m
+    #   :head       e.g. std 80" door + frame/buffers (+ m)
+    #   :sill       e.g. std 30" sill + frame/buffers (+ m)
+    #   :height     any sub surface height, below "head" (+ m)
+    #   :width      e.g. 1.200 m
+    #   :offset     if array (+ m)
+    #   :centreline left or right of base surface centreline (+/- m)
+    #   :r_buffer   buffer between sub/array and right-side corner (+ m)
+    #   :l_buffer   buffer between sub/array and left-side corner (+ m)
```

Comment:
Left/right buffers are distances (in m, always positive numbers) from either left or right edges of the _base_ surface (e.g. wall corners).